src/_locales/de/messages.json

```diff
@@ -68,11 +68,15 @@
   // context menu
   "contextMenuItemConvertSelection": {
     "message": "QR-Code aus Auswahl",
-    "description": "The context menu entry shown for generating QR codes from a slection."
+    "description": "The context menu entry shown for generating QR codes from a selection."
+  },
+  "contextMenuItemConvertLinkSelection": {
+    "message": "QR-Code aus Link erstellen",
```

Comment:
As with the English translation, don't include the word "erstellen" here. It is obvious this add-on does not destroy QR codes.